### ISSUE TITLE
Chrome 7.7.5 - 7.7.6 coming anytime soon?

### ISSUE THREAD

**Issue by ZenFi** [new-xkit/XKit#1372]
Chrome 7.7.5 - 7.7.6 coming anytime soon?

Ver. 7.7.1 is uber old it refused to connect servers when I got it from Chrome Web Store.

Could Firefox version work on Chrome?

(Actually I'm on Opera)

**Comment by nightpool** [comment]
<@341525733396447233> to be clear, we haven't needed to publish any chrome versions since 2015 since there haven't been any framework bugs that need fixing. >90% of New XKit's code is contained within the auto-updating plugin files

So new versions of the base extension are required very infrequently. We've had to publish more Firefox versions because of the instability in their plugin ecosystem—they've adapted two new types of plug-ins in the time we've been publishing new xkit.

7.7.1 is latest chrome software since there haven't been any bugs we've needed to fix in the chromium version of the base framework. It's not a situation where chrome is missing any updates—on the contrary, it means that the chrome extension is much more likely to work since there haven't been any major bugs in the underlying code.
